media/css/firefox/features/article.scss

```diff
@@ -9,6 +9,31 @@ $image-path: '/media/protocol/img';
 @import '~@mozilla-protocol/core/protocol/css/components/callout';
 @import '~@mozilla-protocol/core/protocol/css/components/breadcrumb';
 
+.mzp-c-breadcrumb {
+    padding-top: $spacing-md;
+    padding-bottom: $spacing-md;
+
+    .mzp-c-breadcrumb-item a {
+        color: inherit;
+
+        &:hover {
+            text-decoration: underline;
+        }
+    }
+
+    @media #{$mq-md} {
+        .mzp-c-breadcrumb-list {
+            padding: 8px 64px;
+        }
+    }
+
+    @media #{$mq-lg} {
+        .mzp-c-breadcrumb-list {
+            padding: 8px 96px;
+        }
+    }
```

Comment:
**Suggestion:** This still doesn't completely fix the misalignment, especially on bigger screens. I think a simple solution would be using a `max-width` property with a `margin: 0 auto` declaration:
```suggestion
        .mzp-c-breadcrumb-list {
              max-width: $content-max;
              margin: 0 auto;
        }
```

This essentially is just copying what the site header has in place, and both should successfully align now :)

Reply:
Exactly, I've noticed that every viewport size has its own alignment issues so I'll try to use the same logic the masthead uses to match it not just "close enough", but reliably in all sizes.